spec/Bank.spec.js

```diff
@@ -0,0 +1,99 @@
+const Card = require("../src/Card.js");
+const Bank = require("../src/Bank.js");
+
+describe("", () => {
+  let bank;
+  beforeEach(() => {
+    bank = new Bank();
+    bank.createCard("Tibor");
+  });
+
+  it("deposit money to card", () => {
+    // setUp
+    bank.deposit("Tibor", 400);
+    // execute
+    const result = bank.searchCard("Tibor").balance;
+
+    //verify
+    expect(result).toEqual(1400);
```

Comment:
OK, this is a good test. However, how are you storing the transactions? Would be worth to check if the card has the transaction created as well? Your deposit function might update the balance of the card, but forget to save the transaction, so the test at the moment is inconclusive to me